Fixed awkward phrasing

adipocere-de opened the pull request.

adipocere-de committed:
Fixed awkward phrasing

Morgawr commented on src/Section1/Part1/Lesson13.md:
```diff
@@ -27,7 +27,7 @@ To be a middle schooler <b>and</b> live alone is not possible
 Sometimes distinguishing between the case marker で particle (location, means, etc) and the てform of だ conjunction is not always clear. What matters, however, is that the meaning of the sentence should still be clear. You shouldn’t worry about pinpointing the exact dictionary definition for each particle you see used in a sentence. If you understand the meaning, that’s good enough.
 
-This brings us to では. では is a compound particle. The は here can be considered the topic marker, but the meaning of では together is more than just the sum of its parts. では often marks the circumstances or condition of another state.
+This brings us to では. では is a compound particle. The は here can be considered the topic marker, but the meaning of では together is more than just the sum of its parts. では can be used to say that the statement after it applies to the circumstances of what comes before it.
```
I've been thinking of re-wording this as it still seems a bit confusing. Would you mind writing something like:

"This specific usage of では highlights that the statement that follows applies to the context or circumstances identified by the noun marked by では." 

(I still think it's a bit awkward but probably better?)

Morgawr reviewed (changes requested):
Thanks for the patch and sorry for the late reply. I recommended some changes, would you be able to apply them or do you want me to re-write it myself?

adipocere-de committed:
Fixed awkward phrasing yet again.md

adipocere-de commented:
@Morgawr
I've rewritten the sentence you gave me tried to make it a bit simpler and easier to understand.
I've also added another example sentence that'll hopefully make this use of では clearer.

Let me know if there's anything you don't like & want to be changed.

adipocere-de committed:
Fixed awkward phrasing yet yet again

Morgawr commented:
Thanks!

adipocere-de commented:
Don't mention it.
